conf/config.yaml

```
@@ -1,639 +1,688 @@
+        # for messages).
+        request_retries: 5
+        # Use IPv6 for Telethon connection
+        use_ipv6: false
+
+    # Device info sent to Telegram.
+    device_info:
+        # "auto" = OS name+version.
+        device_model: mautrix-telegram
+        # "auto" = Telethon version.
+        system_version: auto
+        # "auto" = mautrix-telegram version.
+        app_version: auto
+        lang_code: en
+        system_lang_code: en
+
+    # Custom server to connect to.
+    server:
+        # Set to true to use these server settings. If false, will automatically
+        # use production server assigned by Telegram. Set to false in production.
+        enabled: false
+        # The DC ID to connect to.
+        dc: 2
+        # The IP to connect to.
+        ip: 149.154.167.40
+        # The port to connect to. 443 may not work, 80 is better and both are equally secure.
+        port: 80
+
+    # Telethon proxy configuration.
+    # You must install PySocks from pip for proxies to work.
+    proxy:
+        # Allowed types: disabled, socks4, socks5, http, mtproxy
+        type: disabled
+        # Proxy IP address and port.
+        address: 127.0.0.1
+        port: 1080
+        # Whether or not to perform DNS resolving remotely. Only for socks/http proxies.
+        rdns: true
+        # Proxy authentication (optional). Put MTProxy secret in password field.
+        username: ""
+        password: ""
+
 # Python logging configuration.
 #
 # See section 16.7.2 of the Python documentation for more info:
 # https://docs.python.org/3.6/library/logging.config.html#configuration-dictionary-schema
 logging:
-  version: 1
-  formatters:
-    colored:
-      (): mautrix_telegram.util.ColorFormatter
-      format: "[%(asctime)s] [%(levelname)s@%(name)s] %(message)s"
-    normal:
-      format: "[%(asctime)s] [%(levelname)s@%(name)s] %(message)s"
-  handlers:
-    file:
-      class: logging.handlers.RotatingFileHandler
-      formatter: normal
-      filename: /var/log/__APP__/__APP__.log
-      maxBytes: 10485760
-      backupCount: 10
-    console:
-      class: logging.StreamHandler
-      formatter: colored
-  loggers:
-    mau:
-      level: INFO
-    telethon:
-      level: INFO
-    aiohttp:
-      level: INFO
-  root:
-    level: INFO
-    handlers: [file, console]
+    version: 1
+    formatters:
+        colored:
+            (): mautrix_telegram.util.ColorFormatter
+            format: "[%(asctime)s] [%(levelname)s@%(name)s] %(message)s"
+        normal:
+            format: "[%(asctime)s] [%(levelname)s@%(name)s] %(message)s"
+    handlers:
+        file:
+            class: logging.handlers.RotatingFileHandler
+            formatter: normal
+            filename: /var/log/__APP__/__APP__.log
+            maxBytes: 10485760
+            backupCount: 10
+        console:
+            class: logging.StreamHandler
+            formatter: colored
+    loggers:
+        mau:
+            level: DEBUG
```

Is there any specific reason to swtich to DEBUG permanently? Else I suggest we limit logging to INFO, DEBUG might be too noisy.
```suggestion
            level: INFO
```